Bug: 帮助指令报错

### 操作系统

Windows

### Python 版本

3.10.11

### zhenxun_bot 版本

0.2.4

### 适配器

llob

### 协议端

llonebot v4.4.2

### 描述问题

在群聊或私聊发送帮助指令会返回报错

### 复现步骤

在群聊或私聊发送帮助指令

### 期望的结果

_No response_

### 截图或日志（请勿包含敏感信息如密码、令牌等）

12-20 00:46:02 [ERROR] nonebot | Running AlconnaMatcher(type='', module=zhenxun.builtin_plugins.help, lineno=44) failed.
Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\zhenxun_bot\bot.py", line 28, in <module>
    nonebot.run()
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-u0GqDEDQ-py3.10\lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-u0GqDEDQ-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-u0GqDEDQ-py3.10\lib\site-packages\uvicorn\main.py", line 579, in run
    server.run()
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-u0GqDEDQ-py3.10\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-u0GqDEDQ-py3.10\lib\site-packages\nonebot\message.py", line 476, in check_and_run_matcher
    await _run_matcher(
> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-u0GqDEDQ-py3.10\lib\site-packages\nonebot\message.py", line 428, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-u0GqDEDQ-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 850, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-u0GqDEDQ-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 825, in simple_run
    await handler(
  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-u0GqDEDQ-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 94, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
  File "C:\Users\Administrator\Desktop\zhenxun_bot\zhenxun\builtin_plugins\help\__init__.py", line 82, in _
    await create_help_img(session, None)
  File "C:\Users\Administrator\Desktop\zhenxun_bot\zhenxun\builtin_plugins\help\_data_source.py", line 31, in create_help_img
    if help_type.lower() == "html":
AttributeError: 'NoneType' object has no attribute 'lower'

--- a/zhenxun/builtin_plugins/help/_data_source.py
+++ b/zhenxun/builtin_plugins/help/_data_source.py
@@ -27,17 +27,18 @@ async def create_help_img(session: Uninfo, group_id: str | None):
         session: Uninfo
         group_id: 群号
     """
-    help_type: str = base_config.get("type")
-    if help_type.lower() == "html":
-        result = BuildImage.open(await build_html_image(group_id))
-    elif help_type.lower() == "zhenxun":
-        result = BuildImage.open(await build_zhenxun_image(session, group_id))
-    else:
-        result = await build_normal_image(group_id)
-    if group_id:
-        await result.save(GROUP_HELP_PATH / f"{group_id}.png")
-    else:
-        await result.save(SIMPLE_HELP_IMAGE)
+    help_type = base_config.get("type", "").strip().lower()
+
+    match help_type:
+        case "html":
+            result = BuildImage.open(await build_html_image(group_id))
+        case "zhenxun":
+            result = BuildImage.open(await build_zhenxun_image(session, group_id))
+        case _:
+            result = await build_normal_image(group_id)
+
+    save_path = GROUP_HELP_PATH / f"{group_id}.png" if group_id else SIMPLE_HELP_IMAGE
+    await result.save(save_path)
 
 
 async def get_user_allow_help(user_id: str) -> list[PluginType]:

